pkg/plugin/plugin.go

```diff
@@ -559,6 +559,15 @@ func CmdDel(args *skel.CmdArgs) error {
 			}
 			return err
 		})
+		// do the following as per cni spec (i.e. Plugins should generally complete a DEL action
+		// without error even if some resources are missing)
+		if _, ok := err.(ns.NSPathNotExistErr); ok || err == ip.ErrLinkNotFound {
+			if portFound {
```

Comment:
Correct me if I'm wrong, but I believe this is already done on line 541. Is there any reason to call it again?

Reply:
line 541 deletes only interface only from ovs bridge, but interface would still present. so this would try to delete veth pair when link deletion fails on container net namespace.

Reply:
Oh right, thanks